changelog/1.17.0.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "1.17.0"
+description: "Released on 03/17/2021"
+---
+
+### Breaking Changes ❗
+
+- (?) Wsp orgs migration
+
+### Features ✨
+
+- Environment Providers **TODO**
+- Autostart: Autostart allows you to set the time when Coder
+  automatically starts and builds your environments.
+  - web: A new onboarding wizard is available for setting a preferred Autostart
+    time.
+  - web: Visit your account preferences > Autostart to set a time and any
+  environments that should be automatically started. We recommend enabling this
+  feature for the environments you use the most.
+  - web: Autostart action has been added to the Audit Log
+  - web: A new checkbox for opting into Autostart is available on the Create and
+    Edit Environment modals.
```

Comment:
@khorne3 Not sure if best format or not, but it felt like a nice way to present the feature and list all the places it's found.

cc @fuskovic for anything major missed